### PR TITLE
adding taxon, second trial

### PR DESCRIPTION
**Pull request by sabrinatoro**
addresses #3855

### CODE REVIEW

**Comment by sabrinatoro** [comment]
Yoohoo!!! all checks have passed!!! 
Ready for review and merge (as long as you don't see anything weird)
@nicolevasilevsky @matentzn

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -30213,7 +30213,7 @@ xref: UMLS:C0006840 {source="MONDO:equivalentTo", source="NCIT:C26711", source="
 is_a: MONDO:0002041 {source="DOID:1508/inferred", source="ICD10CM:B37", source="MESH:D002177", source="MONDO:Redundant", source="NCIT:C26711"} ! fungal infectious disease
 is_a: MONDO:0002312 {source="DOID:1508", source="MONDO:Redundant"} ! opportunistic mycosis
 intersection_of: MONDO:0005550 ! infectious disease
-intersection_of: disease_has_infectious_agent NCBITaxon:1535326 ! Candida <clade Candida/Lodderomyces clade>
+intersection_of: disease_has_infectious_agent NCBITaxon:1535326
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:30216]: Change suggests taxon is missing in import

**Review by matentzn — COMMENTED** [review]
You removed all the ISBNs?

**Comment by sabrinatoro** [comment]
> You removed all the ISBNs?

I only removed the ISBN-10 (which was not recognized) and kept the ISBN-13,
ISBN-10 was not recognized, and since these IDs referred to the same book, I thought it was ok to remove them.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:30216]: done!

**Review by nicolevasilevsky — APPROVED** [review]

**Pull request merged**